Resources/config/routing.yaml

```diff
@@ -0,0 +1,23 @@
+---
+
+liip_imagine_filter_runtime:
+    path: /media/cache/resolve/{filter}/rc/{hash}/{path}
+    defaults:
+        _controller: '%liip_imagine.controller.filter_runtime_action%'
+    methods:
+        - GET
+    requirements:
+        filter: '[A-z0-9_\-]*'
```

Comment:
If a dash is at the last position of a character match you don't need to escape it, so you could use `[A-z0-9_-]*`.